src/components/Cards/CardsList/CardsList.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useGetHeroesQuery } from '../../../store/heroesApiSlice';
+import CardsItem from '../CardsItem/CardsItem';
+import Spinner from '../../UI/Spinner/Spinner';
+import ErrorMessage from '../../UI/ErrorMessage/ErrorMessage';
+import c from './CardsList.module.scss';
```

Comment:
```suggestion
import s from './CardsList.module.scss';
```
обычно так сокращают, от styles -> s.